Formula/usbutils.rb

```diff
@@ -0,0 +1,44 @@
+class Usbutils < Formula
+  desc "Provides 'lsusb' tool for getting detailed info about USB devices"
+  # Homepage for multiple Linux USB tools, 'usbutils' is one of them.
+  homepage "http://www.linux-usb.org/"
+  # The latest version we can use is 007 (from 2013), because the later
+  # versions depend on 'libudev': part of Systemd and not in Homebrew.
+  url "https://github.com/gregkh/usbutils/archive/refs/tags/v007.tar.gz"
+  sha256 "e5727628ecc0f367df5607862216b71e37eebdf82d6a38c6cc72807f9b2ec159"
+  license any_of: ["GPL-2.0-only", "GPL-3.0-only"]
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  depends_on :macos
+
+  conflicts_with "lsusb", because: "system_profiler-based 'lsusb' is already installed"
+
+  def install
+    system "autoreconf", "--verbose", "--force", "--install"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    # Clear AM_LDFLAGS to remove unsupported "-Wl,--as-needed"
+    # Also, clear/override several other variables, to avoid
+    # building/installing linux-specific utilities that rely on sysfs.
+    system "make", "install", "AM_LDFLAGS=", "SUBDIRS=", "bin_SCRIPTS=",
+                              "man_MANS=lsusb.8"
+  end
+
+  test do
+    # Run --version only. The normal 'lsusb' run may exit
+    # with 1 if executed on a Virtual Machine.
+    system "#{bin}/lsusb", "--version"
```

Comment:
Exiting with 1 is preferable to a version test that doesn't execute a real code path.

Reply:
Okay, I'll change it.